deploy/helm/sumologic/conf/setup/main.tf

```diff
@@ -1,6 +1,5 @@
 terraform {
   required_providers {
     sumologic  = "~> 2.3.0"
-    kubernetes = "~> 1.11.3"
```

Comment:
why?

Reply:
ouch the other way around, let me fix that